fenix/app/src/main/res/values/strings.xml

```diff
@@ -246,6 +246,8 @@
     <string name="default_locale_text">Follow device language</string>
     <!-- Placeholder text shown in the search bar before a user enters text -->
     <string name="locale_search_hint">Search language</string>
+    <!-- Content description for the translate page toolbar button that opens the translations dialog when no translation has occurred. -->
+    <string name="browser_toolbar_translate">Translate page</string>
```

Comment:
Let's move this below [L236](https://github.com/mozilla-mobile/firefox-android/pull/5305/files#diff-b24d2928f9f4594421c030b62d3e5e070c9fd5600c871ff9a8c24a518fd02f96L236) since we have actually have a Browser Toolbar section and you probably want to add a new line to space out the sections.

Reply:
Thanks, yeah totally didn't see ` <!-- Locale Settings Fragment -->` stuck between the browser toolbar section and what I added. 🔎  

Moved it up and made the spacing between sections more clear.